src/textual/_border.py

```diff
@@ -135,7 +141,12 @@ def __init__(
             (bottom, bottom_color),
             (left, left_color),
         ) = edge_styles
-        self._sides = (top or "none", right or "none", bottom or "none", left or "none")
+        self._sides: tuple[EdgeType, ...] = (
```

Comment:
Added a few type hints here and there to help me understand what some of our variables represent... 
And I guess it will also be useful to have MyPy/PyRight double check these things too :slightly_smiling_face:

Reply:
Not sure this is the most precise type, `tuple[EdgeType, ...]` means a tuple of any number of `EdgeType` instances, where I think that it may be `tuple[EdgeType, EdgeType, EdgeType, EdgeType]`, i.e. a tuple of exactly 4.